environment/plaintext/docker-compose.yml

```diff
@@ -7,12 +7,15 @@ services:
     hostname: zookeeper
     container_name: zookeeper
     restart: always
+    volumes:
+      - ./jmx-exporter:/usr/share/jmx_exporter/
```

Comment:
I can revert it back to other/monitoring-demo. The though process was to so users didn't have to go into different directories if they want to test things out. If you want to modify your local JMX metrics and you need to make docker-compose changes, you can implement it all in one directory. If you want to prevent duplicates I can undo this just didn't want users going to multiple locations to find the info they are looking for.

Reply:
ok let's leave it as is for now.
In the future, I'd like to do some re-org with a "reusables" folder containing these kind of things that are re-used over the playground